view_all_set.php

```diff
@@ -374,11 +94,6 @@
 	$f_per_page = config_get( 'default_limit_view' );
 }
 
-# combine sort settings
-#  (f_sort overrides f_sort_1 if set to keep old sorting code working in view_all_bug_inc)
-$f_sort = ( ( $f_sort_d != '' ) ? $f_sort_d : $f_sort_0 ) . ( ( $f_sort_1 != '' ) ? ',' . $f_sort_1 : '' );
-$f_dir = ( ( $f_dir_d != '' ) ? $f_dir_d : $f_dir_0 ) . ( ( $f_dir_1 != '' ) ? ',' . $f_dir_1 : '' );
-
```

Comment:
fixed by using filter_gpc_get() on all those situations
most of the view_all_set operatios, defined by "type" request parameter, can be generalized to do a filter_gpc_get